apps/melding-form/src/app/(map)/locatie/kies/_components/Map.tsx

```diff
@@ -52,6 +52,38 @@ export const Map = ({ coordinates, showAssetList, setCoordinates }: Props) => {
       ],
     })
 
+    const crosshair = document.getElementById('crosshair')!
+
+    // Set crosshair to show where the marker is set with a keyboard
```

Comment:
2 things can possibly happen on keydown, and this comment doesn't really describe them. I'd add a comment above both `if` statements, and remove this one.

Reply:
ha, I forgot to push the last commit where I changed the comments;)